### PR TITLE
Topic/mnaveau/ci

### PR DESCRIPTION
**Pull request by MaximilienNaveau**
Introduce a simple CI that checks the Eigen conversions
cf: https://github.com/stack-of-tasks/dynamic_graph_bridge_msgs/tree/master/.github/workflows

### CODE REVIEW

**Comment by hrp2-14** [comment]
Hi ! This project doesn't usually accept pull requests on the main branch.
If this wasn't intentionnal, you can change the base branch of this PR to devel
(No need to close it for that). Best, a bot.

File: .github/workflows/ci_ubuntu20_04_ros1_noetic.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+on:
+  push:
+    branches:
+    - master
+    - devel
+  pull_request:
+    branches:
+    - master
+    - devel
```
Comment (nim65s) [inline .github/workflows/ci_ubuntu20_04_ros1_noetic.yml:11]:
```suggestion
on: [push, pull_request]
```

File: .github/workflows/ci_ubuntu18_04_ros1_melodic.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+on:
+  push:
+    branches:
+    - master
+    - devel
+  pull_request:
+    branches:
+    - master
+    - devel
```
[the remark this answers is not in the document]
Comment (MaximilienNaveau) [reply .github/workflows/ci_ubuntu18_04_ros1_melodic.yml:11]: fair!

**Comment by MaximilienNaveau** [comment]
Waiting for the CI to converge in my fork before merging this

**Pull request merged**